### ISSUE TITLE
Number field in multipart request

### ISSUE THREAD

**Issue by vaiil** [oazapfts/oazapfts#622]
I have the following schema:

```yaml
  /private-api/projects/{projectId}/files/upload:
    post:
      operationId: uploadFiles
      summary: Upload file
      parameters:
        - $ref: '#/components/parameters/projectId'
      requestBody:
        required: true
        content:
          multipart/form-data:
            schema:
              type: object
              required: [ entityType, entityId, fieldId, file ]
              properties:
                entityType:
                  type: string
                  enum: [ 'product', 'offer' ]
                entityId:
                  type: number
                  format: int64
                fieldId:
                  type: number
                  format: int64
                file:
                  type: string
                  format: binary
```

Generated api:

```ts
/**
 * Upload file
 */
export function uploadFiles(
  projectId: number,
  body: {
    entityType?: 'product' | 'offer';
    entityId?: number;
    fieldId?: number;
    file?: Blob;
  },
  opts?: Oazapfts.RequestOpts,
) {
  return oazapfts.fetchJson<{
    status: 200;
    data: File;
  }>(
    `/private-api/projects/${encodeURIComponent(projectId)}/files/upload`,
    oazapfts.multipart({
      ...opts,
      method: 'POST',
      body,
    }),
  );
}
```

When I use this function, I got entity fields converted to blob:

```txt
------WebKitFormBoundarybwyXgtcDY9O04C2R
Content-Disposition: form-data; name="entityType"

product
------WebKitFormBoundarybwyXgtcDY9O04C2R
Content-Disposition: form-data; name="entityId"; filename="blob"
Content-Type: application/json

1
------WebKitFormBoundarybwyXgtcDY9O04C2R
Content-Disposition: form-data; name="fieldId"; filename="blob"
Content-Type: application/json

13
------WebKitFormBoundarybwyXgtcDY9O04C2R
Content-Disposition: form-data; name="file"; filename="test.test.xlsx"
Content-Type: application/vnd.openxmlformats-officedocument.spreadsheetml.sheet


------WebKitFormBoundarybwyXgtcDY9O04C2R--
```

There is a transformation of request fields to blob in [runtime.ts](https://github.com/oazapfts/oazapfts/blob/774d9de246abc5b081e544253d73a072befcddfb/packages/runtime/src/runtime.ts#L153C2-L153C66):

```ts
if (typeof value === "string" || value instanceof Blob) {
  data.append(name, value);
} else {
  data.append(
    name,
    new Blob([JSON.stringify(value)], { type: "application/json" }),
  );
}
```

What about adding `typeof value === "number"` to condition?

**Comment by vaiil** [comment]
Add fix in #623

**Comment by Xiphe** [comment]
fixed by https://github.com/oazapfts/oazapfts/pull/623
